style.css

```diff
@@ -100,6 +107,367 @@ ul li {
+
+.multi-project .project-tag p {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 11px;
+  line-height: 20px;
+  color: #091e42;
+  background: #fff;
+  padding: 10px 12px;
+  border: 1px solid #8993a4;
+}
+
+.multi-project .project-view {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 12px;
+  gap: 10px;
+  margin-top: 8px;
+  width: 134px;
+  height: 48px;
+  background: #ff6b00;
+}
+
+.project-view a {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  width: 200px;
+  height: 24px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #fff;
+  cursor: pointer;
+}
+
+/* all projects multi section end */
+
+/* single projects start */
+
+.my-main-works .single-project {
+  width: 100%;
+  height: 386px;
+}
+
+.my-main-works .single-project .project-image {
+  width: 100%;
+  height: 85%;
+  background: url('./images/project_image.png');
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  display: flex;
+  flex-direction: column;
+  padding-left: 16px;
+  padding-right: 16px;
+  padding-top: 80px;
+  gap: 15px;
+  overflow: hidden;
+}
+
+.my-main-works .single-project .project-image .single-title {
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff;
+}
+
+.my-main-works .single-project .project-image .single-description {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #fff;
+}
+
+.my-main-works .single-project .project-image .single-language {
+  display: flex;
+  gap: 12px;
+  flex-direction: row;
+}
+
+.my-main-works .single-project .project-image .single-language li {
+  padding: 10px 12px;
+  background: rgba(255, 255, 255, 0.24);
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: #fff;
+  text-align: center;
+}
+
+.single-project .view-project {
+  width: 100%;
+  height: 48px;
+  background-color: #ff6b00;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.single-project .view-project #view-project-id {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #fff;
+}
+
+/* single project section end */
+
+/* about me section start */
+.about-me {
+  width: 100%;
+  height: 1901px;
+  background-image: url('./images/about-me.png');
+  background-repeat: no-repeat;
+  background-position: right center;
+  background-position-y: 350.6px;
+  display: flex;
+  flex-direction: column;
+  background-color: #1c1a19;
+  padding-left: 24px;
+  padding-right: 24px;
+  padding-top: 120px;
+  gap: 12px;
+}
```

Comment:
- [ ] Kindly use more gaps between the `My Recent Works` and `About Me` section so that it matches the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471&t=3CbGHNhPHWBiQT4V-0).
  - ***Left one is your design and the Right one is Figma design***
  ![Screenshot from 2023-01-22 00-23-04](https://user-images.githubusercontent.com/60083437/213881563-d506e36d-3e09-473c-bc69-064f5a8ebce2.png)